From 45927b8c6bae1d80edf56e9b070c55b526226548 Mon Sep 17 00:00:00 2001
From: Joakim Zhang <qiangqing.zhang@nxp.com>
Date: Thu, 23 Aug 2018 09:03:31 +0800
Subject: [PATCH 4449/5242] MLK-19166-2 can: flexcan: enable flexcan support
 for arm64

commit  cfb468c7086762328d759bfacdf4eb20564beebd from
https://source.codeaurora.org/external/imx/linux-imx.git

Modify Kconfig file to enable flexcan support for arm64

Signed-off-by: Joakim Zhang <qiangqing.zhang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/can/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/can/Kconfig b/drivers/net/can/Kconfig
index 2cb7598..55acc5b 100644
--- a/drivers/net/can/Kconfig
+++ b/drivers/net/can/Kconfig
@@ -90,7 +90,7 @@ config CAN_AT91
 
 config CAN_FLEXCAN
 	tristate "Support for Freescale FLEXCAN based chips"
-	depends on ARM || PPC
+	depends on ARM || ARM64 || PPC
 	---help---
 	  Say Y here if you want to support for Freescale FlexCAN.
 
-- 
1.7.9.5

